lib/galaxy/tools/verify/__init__.py

```diff
@@ -138,7 +139,7 @@ def get_filename(filename):
         except AssertionError as err:
             errmsg = '%s different than expected, difference (using %s):\n' % (item_label, compare)
             errmsg += "( %s v. %s )\n" % (local_name, temp_name)
-            errmsg += str(err)
+            errmsg += text_type(err)
```

Comment:
I'm going to also use `unicodify` here instead of `text_type`, shout if you think that's not necessary.